src/clincoded/static/components/provisional_classification/approval.js

```diff
@@ -182,11 +182,12 @@ const ClassificationApproval = module.exports.ClassificationApproval = createRea
      */
     handlePreviewApproval() {
         const affiliationId = this.props.affiliation ? this.props.affiliation.affiliation_id : null;
+        const provisionalAffiliation = this.props.provisional.affiliation ? this.props.provisional.affiliation : null;
```

Comment:
Is `this.props.provisional` guaranteed to be defined? I didn't notice in the original PR, but it could be safer to add a check for it.

`const provisionalAffiliation = this.props.provisional && this.props.provisional.affiliation ? this.props.provisional.affiliation : null;`

Reply:
Going to add that check and push!